experimental/examples/bubblebench/editable-shared-tree/src/appState.ts

```diff
@@ -2,25 +2,25 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { IAppState, IClient, makeBubble, randomColor } from "@fluid-example/bubblebench-common";
-import { brand, EditableField, FieldKey } from "@fluid-internal/tree";
-import { Client } from "./client";
-import { ClientTreeProxy } from "./schema";
+import { IAppState, makeBubble, randomColor } from "@fluid-example/bubblebench-common";
+import { brand, FieldKey } from "@fluid-internal/tree";
+import { ClientWrapper } from "./client";
+import { ClientsField, Client, FlexClient, FlexBubble } from "./schema";
 
 export class AppState implements IAppState {
 	static clientsFieldKey: FieldKey = brand("clients");
-	readonly localClient: Client;
+	readonly localClient: ClientWrapper;
 
 	constructor(
-		private readonly clientsSequence: ClientTreeProxy[] & EditableField,
+		private readonly clientsSequence: ClientsField,
 		public width: number,
 		public height: number,
 		numBubbles: number,
 	) {
 		clientsSequence[clientsSequence.length] = this.createInitialClientNode(
 			numBubbles,
-		) as ClientTreeProxy;
-		this.localClient = new Client(clientsSequence[clientsSequence.length - 1]);
+		) as Client;
+		this.localClient = new ClientWrapper(clientsSequence[clientsSequence.length - 1]);
```

Comment:
I see.  You're rotating the naming convention:

- *Proxy -> *
- * -> *Wrapper

Probably a net win if using Bubblebench as a SharedTree sample but introduces an inconsistency when comparing Bubblebench implementations.

I'm ambivalent.

Reply:
I want to get "Proxy" out of the names since it an implementation detail thats supposed to be private, so I think this is a win.